fpm/src/fpm_command_line.f90

```diff
@@ -29,11 +29,14 @@ module fpm_command_line
     logical                      :: with_test=.false.
     logical                      :: with_lib=.true.
     logical                      :: backfill=.true.
+    logical                      :: verbose=.true.
```

Comment:
Brilliant! I've been wanting a `verbose` flag for some time to help with development. Since it is likely to be needed for all commands, I would suggest having `verbose` as a component of the overall base type `fpm_cmd_settings`.